blog/_posts/2018-09-17-team-sectumsempra.md

```diff
@@ -0,0 +1,155 @@
+
+In addition to giving us great memories to cherish, it also marked the beginning of our Open Source journeys. We have been collaborating from four different time zones across the world: IST, EET, EDT, PDT. That’s the best part. It was my first time collaborating with people from overseas. And, it has been a great one.
+
+Along with a lot of amazing and thought provoking things we worked on this summer, one thing I learnt during the program that I wish to share with Y’all: It is okay if you don’t know something in advance, everybody starts as a beginner. It’s more about how willing you’re to learn after that. There have been a lot of ‘trivial’ things (although not-so-trivial for me) that I got to learn this summer. There have been times when I used to feel very small about not knowing something as small as “Git workflow” which, apparently, everybody on the Chapel channel was hella great at. But you know what, chuck those thoughts away and ask for help if you’re feeling stuck.
+
+![Alt Text](https://github.com/avneet14027/images/blob/master/2018-09-17-giphy2.gif)
+<div class="image-credits">Source: Giphy</div>
+
+I promised myself that I am going to make this summer that best summer of my undergraduate for me. Won’t leave any stone unturned. And, I did. No matter how silly and tiring the tasks were, I sailed through them and did my best.
+
+#### My advice to RGSoC '19 aspirants
+
+- **Network**: Meet new people and talk to as many people as you can. You’ll never know
+who you’ll run into one day. It might just be your life mentor, your next recruiter, your
+potential thesis advisor, or maybe your “he is the one” person. :P
+Go to conferences, meetups, workshops, attend short talks that your university arranges
+for you. Talk to the speakers over there, ask for their current running projects and ask
+them if they can look at your profile and can recommend you at their company or can
+mentor you. Sounds far-fetched, right? Believe us, this works. It has worked for us, at
+least. :D
+
+- **Communicate**: Be it the application period or the RGSoC internship period, make sure
+you don’t leave any communication gaps. Be clear with what you want. Be concise while
+asking for favour or help, but ASK! Don’t shy away.
+
+- **Set the bars high for yourselves**: In our experience, we have observed that when it
+comes to career, expectation bar is embarrassingly low for women. And this is majorly
+because why we don’t see ourselves being able to match men. Think that you’ll achieve
+big, and so you will!
+
+Kate Winslet just puts it right:
+
+[![Kate Winslet advice to women](http://img.youtube.com/vi/DeRcIf4sZiM/0.jpg)](http://www.youtube.com/watch?v=DeRcIf4sZiM "Kate Winslet advice to women")
```

Comment:
Hi @avneet14027 ! You added bunch of more images/GIFs in the file and seem to have forgot to add image credits. Please add them here.

Reply:
Hi @nerdyvaishali , I have added the image credits now.
Thanks,
Avneet